wiki_docs/Create How_GTM_works

```diff
@@ -0,0 +1,47 @@
+User can add or manage the AdWords, Google analytics, Flood-light and non-Google tags using the GTM.
+
+How it works
+  Followings are main components of the GTM:
```

Comment:
There are a few main components of GTM: (1.) Account (2.) Container (3.)Tags (4.) Triggers (5.) Variables (6.) Folders

@amitkumarj441: I believe that the version of GTM we are using no longer uses "rules" or "macros". They are called "triggers" and "variables" now.